README.md

```diff
@@ -268,6 +269,84 @@ architecture model:
 See the included example for usage of some those properties in the
 [C4 architecture model example](https://github.com/avisi-cloud/structurizr-site-generatr/blob/main/docs/example/workspace.dsl#L159).
 
+## Running as a github action
+
+To generate the site, for a set of branches you can use the builtin [action.yml](action.yml) directly from your own worflow:
+```yaml
+jobs:
+  ...
+  steps:
+    - name: Build docs
+      uses: avisi-cloud/structurizr-site-generatr@action
+      with:
+        repository: https://github.com/MY_ORG/MY_REPO.git
+        default-branch: develop
+        branches: main,develop
+        workspace-file: path/to/workspace.dsl
+        token: ${{ secrets.GH_ACCESS_TOKEN }}
+```
+
+The action will use the github workspace, rather than `/var/model`.
+Additionally, you will need to generate a personal acess token with, at least, read access to your repository.
```

Comment:
You don't always need a personal access token. This repository is public, so if you provide Structurizr Site Generatr with a `https://` URL, a PAT isn't needed. See for example https://github.com/avisi-cloud/structurizr-site-generatr/blob/main/.github/workflows/release.yml#L165.